apps/native-component-list/package.json

```diff
@@ -36,8 +36,8 @@
     "@expo/react-native-action-sheet": "^3.8.0",
     "@expo/styleguide-base": "^1.0.1",
     "@react-native-async-storage/async-storage": "1.23.1",
-    "@react-native-community/datetimepicker": "8.0.0",
-    "@react-native-community/netinfo": "11.3.1",
+    "@react-native-community/datetimepicker": "8.2.0",
+    "@react-native-community/netinfo": "11.4.1",
```

Comment:
is netinfo bump intentional in this pr?

Reply:
Sorry, for this 🤦 something seems wrong with my git